Formula/gst-python.rb

```diff
@@ -32,15 +33,22 @@ class GstPython < Formula
   end
 
   def install
-    mkdir "build" do
-      system "meson", *std_meson_args, ".."
-      system "ninja", "-v"
-      system "ninja", "install", "-v"
-    end
+    python = "python3.10"
+    site_packages = prefix/Language::Python.site_packages(python)
+
+    # This shouldn't be needed, but this fails to link with libpython3.10.so.
+    # TODO: Remove this when `python@3.10` is no longer keg-only.
+    ENV.append "LDFLAGS", "-Wl,-rpath,#{Formula["python@3.10"].opt_lib}" if OS.linux?
```

Comment:
Seems odd. I wonder if there is a `brew` bug. I downloaded previous run's bottle and it is interesting that one library (`lib/python3.10/site-packages/gi/overrides/_gi_gst.cpython-310-x86_64-linux-gnu.so`) has `python@3.10` in RPATH but the other (`lib/gstreamer-1.0/libgstpython.so`) doesn't.

The only difference between two RPATHS is `@@HOMEBREW_PREFIX@@/opt/python@3.10/lib`.

Reply:
Would help to be able to find the compiler/linker invocation when these objects are created, but Linux doesn't upload logs properly, unfortunately.